src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,87 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+
+@Entity
+public class User {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  @Column(nullable = false)
+  private String username;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private LocalDate birthdate;
+
+  @ManyToMany(cascade = CascadeType.ALL)
+  @JoinTable(name = "books_users",
```

Comment:
Only necessary if you want custom table and column names, given a ManyToMany infers a JoinTable already.

Reply:
👌